src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html

```diff
@@ -16,34 +16,43 @@
 
 <div class="row">
     <div class="{% if settings.bank_details_type == "sepa" %}col-md-6{% else %}col-md-12{% endif %} col-xs-12">
-        <p>
             {% if settings.bank_details_type == "sepa" %}
-                <strong>{% trans "Account holder" %}:</strong> {{ settings.bank_details_sepa_name }}<br>
-                <strong>{% trans "IBAN" %}:</strong> {{ settings.bank_details_sepa_iban|ibanformat }}<br>
-                <strong>{% trans "BIC" %}:</strong> {{ settings.bank_details_sepa_bic }}<br>
-                <strong>{% trans "Bank" %}:</strong> {{ settings.bank_details_sepa_bank }}<br>
+                <dl class="dl-horizontal">
+                    <dt>{% trans "Account holder" %}:</dt><dd>{{ settings.bank_details_sepa_name }}</dt>
+                    <dt>{% trans "IBAN" %}:</dt><dd>{{ settings.bank_details_sepa_iban|ibanformat }}</dt>
+                    <dt>{% trans "BIC" %}:</dt><dd>{{ settings.bank_details_sepa_bic }}</dt>
+                    <dt>{% trans "Bank" %}:</dt><dd>{{ settings.bank_details_sepa_bank }}</dt>
+                {% if details %}
+                </dl>
+                {% endif %}
             {% endif %}
             {% if details %}
-                {{ details|linebreaksbr }}<br>
+                {{ details|rich_text }}
```

Comment:
Good catch! I actually just changed it because of multiple line breaks being „forbidden“ (they were marked as an „Verstoß“ in A11y-testing) and rich_text converts multiple line breaks to <p> elements. Is the a filter that das linebreaksbr and turns multiple linebreaks into <p> and nothing else?

Reply:
As far as I can remember it actually wasn’t my intention to add markdown-support, but to add support for paragraphs – which `|linebreaks` does just fine. I removed `rich_text` and changed it to `linebreaks`. Changed it in checkout_payment_form.html as well.